web/src/features/charts/hooks/useOriginChartData.ts

```diff
@@ -180,14 +180,14 @@ interface ValuesInfo {
 function getValuesInfo(
   historyData: ZoneDetail[],
   displayByEmissions: boolean,
-  timeAggregate: string
+  timeRange: string
 ): ValuesInfo {
   const maxTotalValue = d3Max(historyData, (d: ZoneDetail) =>
     displayByEmissions
       ? getTotalEmissionsAvailable(d, true)
       : getTotalElectricityAvailable(d, true)
   );
-  const isHourly = timeAggregate === TimeAverages.HOURLY;
+  const isHourly = timeRange === TimeRange.H24;
```

Comment:
```suggestion
  // TODO: Make this use the atom as a single source of truth.
  const isHourly = timeRange === TimeRange.H24;
```

How does this work with the 72 hour view? That is also hourly?

Reply:
Moving this up to a hook!